### ISSUE TITLE
About reading data

### ISSUE THREAD

**Issue by MorganCZY** [JeremyCCHsu/vae-npvc#6]
In “main.py”, training data are got by " image, label = read(***)" for once. But I found the function "read()" in analyzer.py can only return a batch-size samples and there is no circulation for getting other samples.  Thus, only a batch-size samples are used to train the network. Do I understand your code correctly? If yes, is it the right way to read training data?

**Comment by JeremyCCHsu** [comment]
The way Tensorflow (older version) does mini-batching is not very intuitive due to its symbolic nature. The batching is actually over the whole dataset. Please consult the [API ](https://www.tensorflow.org/api_docs/python/) for more information.
